jax/lib/xla_bridge.py

```diff
@@ -120,19 +120,28 @@ def _get_local_backend(platform=None):
   return backend
 
 
+register_backend('xla', _get_local_backend)
+
+
+# memoize the TPU driver to be consistent with xla_client behavior
+_tpu_backend = None
+
 def _get_tpu_driver_backend(platform):
   del platform
```

Comment:
Something I just realized is that this will happily return the TPU backend even if you pass in 'cpu' or 'gpu' for the platform. We could potentially raise an error in this case, but I think we should leave it for now.